landscaper/common/generate/blueprint/generate.go

```diff
@@ -67,15 +69,41 @@ type BlueprintSchema struct {
 	Items []BlueprintSchema `json:"items,omitempty"`
 }
 
+// ExportExecutions are the export executions of blueprint corresponding to the exported fields
+type ExportExecutions struct {
+	// Exports are the exported fields
+	Exports map[string]string `json:"exports,omitempty"`
+}
+
+var (
+	// defaultExportExecutionFileName is the default name for the export execution file name
+	defaultExportExecutionFileName = "deploy-execution.yaml"
+	// defaultContainerDeployerName is the default name used for the container deployer when rendering the export executions
+	defaultContainerDeployerName = "default"
+	// typeTarget is the type of field (either in the import or export section of the blueprint)
+	typeTarget = "target"
+)
```

Comment:
You could make them const and use `pointer.String` (as it is not possible to get address of const).

```suggestion
const (
	// defaultExportExecutionFileName is the default name for the export execution file name
	defaultExportExecutionFileName = "deploy-execution.yaml"
	// defaultContainerDeployerName is the default name used for the container deployer when rendering the export executions
	defaultContainerDeployerName = "default"
	// typeTarget is the type of field (either in the import or export section of the blueprint)
	typeTarget = "target"
)
```